src/components/ha-clickable-list-item.ts

```diff
@@ -0,0 +1,66 @@
+import { style } from "@material/mwc-list/mwc-list-item-css";
+import { ListItemBase } from "@material/mwc-list/mwc-list-item-base";
+import { css, CSSResult, customElement, property } from "lit-element";
+import { html } from "lit-html";
+
+@customElement("ha-clickable-list-item")
+export class HaClickableListItem extends ListItemBase {
+  public href?: string;
+
+  public disableHref = false;
+
+  // property used only in css
+  @property({ type: Boolean, reflect: true }) public rtl = false;
+
+  public render() {
+    const r = super.render();
+    const href = this.href ? `/${this.href}` : "";
+
+    return html` ${this.renderRipple()}
+    ${this.disableHref
+      ? html`<a aria-role="option">${r}</a>`
+      : html`<a aria-role="option" href=${href}>${r}</a>`}`;
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      style,
+      css`
+        :host {
+          padding-left: 0px;
+          padding-right: 0px;
+        }
+
+        :host([rtl]) {
+          padding-left: var(--mdc-list-side-padding, 0px);
+          padding-right: var(--mdc-list-side-padding, 0px);
+        }
```

Comment:
Why only for rtl?

Reply:
Actually, I'm not sure why this is here.. now, in localhost, it actually seems to mess up rtl styling. 🤷 Removing.